bitnami/postgresql-ha/values.yaml

```diff
@@ -1550,10 +1583,17 @@ metrics:
   ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
   ## @param metrics.podSecurityContext.enabled Enable security context for PostgreSQL Prometheus exporter
   ## @param metrics.podSecurityContext.runAsUser User ID for the PostgreSQL Prometheus exporter container
+  ## @param metrics.podSecurityContext.runAsGroup Group ID for the PostgreSQL Prometheus exporter container
+  ## @param metrics.podSecurityContext.runAsNonRoot Set PostgreSQL Prometheus exporter container's Security Context runAsNonRoot
+  ## @param metrics.podSecurityContext.seccompProfile.type Set PostgreSQL Prometheus exporter  container's Security Context seccompProfile
   ##
   podSecurityContext:
     enabled: true
     runAsUser: 1001
+    runAsGroup: 0
+    runAsNonRoot: true
+    seccompProfile:
```

Comment:
Should we include the following container security context constraints here? Is there any reason to not include them for this container?

```
    allowPrivilegeEscalation: false
    capabilities:
      drop:
        - ALL
```

Reply:
I believe that was an oversight on my end.  I'll get those added and the branch rebased.

Reply:
You are right, I did not notice this one was for the PodSecurityContext. 👍